### PR TITLE
Getting Started Guide Bxmsdoc 1321 master

### PR DESCRIPTION
**Pull request by michelehaglund**
This Jira work covers the LA GSG chapters 6, 7, and 8. If time permits, I will add additional screen captures when a fully functioning build is available. For now, please focus on the correctness and formatting of the procedures.

Original doc Jira: https://issues.jboss.org/browse/BXMSDOC-1321
Rendered output: http://file.rdu.redhat.com/~mhaglund/BXMSDOC-1321/

### CODE REVIEW

File: docs/product-getting-started-guide/src/main/asciidoc/task_business-rules.adoc [hunk 1]
```diff
@@ -9,102 +9,108 @@ The following chapter instructs you how to create business rules in {PRODUCT} us
 [float]
 == Prerequisites
 
-* Finished data object in {PRODUCT}. For further information, see <<_defining_a_data_model>>.
+Finished data object in {PRODUCT}. For further information, see <<_defining_a_data_model>>.
 
 == Creating Guided Rules
 
 To create a guided rule:
```
Comment (gsheldon) [inline docs/product-getting-started-guide/src/main/asciidoc/task_business-rules.adoc]: I think starting this with a full sentence "Use the following procedure to create a guided rule:" would be better. I vaguely remember a discussion with a translator about this sort of sentence fragment being an issue for them. I'm guilty of applying this inconsistently myself, so feel free to ignore me.

File: docs/product-getting-started-guide/src/main/asciidoc/task_business-rules.adoc [hunk 2]
```diff
@@ -9,102 +9,108 @@ The following chapter instructs you how to create business rules in {PRODUCT} us
...
+. Click the *There is an Application* label and select *Any of (Or)* from *Multiple field constraints*.
+. Click the *There is an Application with: any of the following:* label , then select *downpayment* from *Add a restriction on a field*.
+. Click *--- please choose ---*, then select *equal to*.
+. Click image:6191.png[], select *Literal value*, and enter `0`.
+. Click the *There is an Application with: any of the following:* label, then from the *Add a restriction on a field* select *downpayment*.
+. Click *--- please choose ---*, then select *greater than or equal to*.
+. Click image:6191.png[], then click *Expression editor*.
+. From the *Choose...* menu, select *prop*.
+. From the *Choose...* menu, select *salesprice*.
+. Click image:6176.png[] next to the *THEN* label. select *Insert fact ValidationErrorDO...*, and click *+Ok*.
```
Comment (gsheldon) [inline docs/product-getting-started-guide/src/main/asciidoc/task_business-rules.adoc]: s/select/Select

File: docs/product-getting-started-guide/src/main/asciidoc/task_business-rules.adoc [hunk 3]
```diff
@@ -9,102 +9,108 @@ The following chapter instructs you how to create business rules in {PRODUCT} us
...
+. Click the *Insert ValidationErrorDO* label and select *error* from the *Add field* pull-down menu.
+. Click image:6191.png[], then click *Literal value*.
+. In the *error* field, enter: `Down payment cannot be 0, greater than, or equal to the property sale price.`.
+. Click image:6176.png[] next to the *THEN* label. Select *Add free form DRL* and click *+Ok*.
+. Enter the following in the _Add free form DRL.._ text box:
```
Comment (gsheldon) [inline docs/product-getting-started-guide/src/main/asciidoc/task_business-rules.adoc]: needs another dot in the _Add free form DRL.._ .

File: docs/product-getting-started-guide/src/main/asciidoc/task_business-rules.adoc [hunk 4]
```diff
@@ -9,102 +9,108 @@ The following chapter instructs you how to create business rules in {PRODUCT} us
...
-
-Click the *process-app-start* label to return to the *Assets* view of the project. 
+. Click *(show options...)*.
+. Click image:6176.png[] next to the *(options)* label to open the *Add an option to the rule* window.
+. Select *ruleflow-group* from the *Attribute* drop-down menu and enter *validation* in the *ruleflow-group* field.
+. Click *Save*, then click *Save* to confirm your changes.
 
 == Creating Guided Decision Tables
 
 To create a guided decision table:
```
Comment (gsheldon) [inline docs/product-getting-started-guide/src/main/asciidoc/task_business-rules.adoc]: As above (sentence fragment).

File: docs/product-getting-started-guide/src/main/asciidoc/task_business-rules.adoc [hunk 5]
```diff
@@ -9,102 +9,108 @@ The following chapter instructs you how to create business rules in {PRODUCT} us
...
-.. From *Conditions*, select *saleprice*, then enter the following: 
++
+. From *Available patterns*, select *Property*,  then from *Available fields*, select *saleprice:Whole number (interger)* and then move it to the *Conditions* section.
+. From *Conditions*, select *saleprice*, then enter:
 +
 * *Column header (description)*: `Sale Price (Less Than)`
 * *Operator*: `less than`
-.. From *Available fields*, select *age*, and place it into *Conditions*. 
-.. From *Conditions*, select *age*, then enter the following: 
+. From *Available fields*, select *age:Whole number (interger)*, and move it in to the *Conditions* section.
```
Comment (gsheldon) [inline docs/product-getting-started-guide/src/main/asciidoc/task_business-rules.adoc]: Can this just be "move it to the *Conditions* section"? I'm not actually sure if that makes sense without seeing it, ignore me if it doesn't.

File: docs/product-getting-started-guide/src/main/asciidoc/task_business-rules.adoc [hunk 6]
```diff
@@ -9,102 +9,108 @@ The following chapter instructs you how to create business rules in {PRODUCT} us
...
-. Click *Decision table* -> *New column*, select *include advanced options*, select *Add a new Attribute column*, then click *Next*.
+. Click *Save*, then *Save*, to confirm your changes.
+. Click *Insert > Append row* and click *Save*.
+. Click *Add Column*.
+. From the *Add a new column* window, select *New Column*.
+. Select *Include advanced options*, then select *Add a new Attribute column*.
 . Select *Ruleflow-group*, then click *Finish*.
-. Click *Insert* -> *Append row*. Repeat this step once.
-. Fill out the table as follows:
+. Fill out the table as shown below:
```
Comment (gsheldon) [inline docs/product-getting-started-guide/src/main/asciidoc/task_business-rules.adoc]: IBM - "below" do not use to indicate a relative location in a document. You can use "following" to refer to a table though, so perhaps "Fill out the table as shown in the following example:" or whatever.

File: docs/product-getting-started-guide/src/main/asciidoc/task_data-model.adoc [hunk 7]
```diff
@@ -62,5 +62,5 @@ In the `Application` data model:
 * *Type*: `Mortgage Property (org.jboss.example.Property)`
 
 . Click *Create*.
-. Click *Save*, then click *Save* to confirm, to save your data object.
+. Click *Save*, then click *Save* to confirm, to confirm your changes.
```
Comment (gsheldon) [inline docs/product-getting-started-guide/src/main/asciidoc/task_data-model.adoc]: remove the comma and second "to confirm".

File: docs/product-getting-started-guide/src/main/asciidoc/task_deploying-application.adoc [hunk 8]
```diff
@@ -19,9 +19,9 @@ To deploy an application in {PRODUCT}:
 . Log into Business Central.
 . Click *Menu* -> *Projects*.
 . Click on a project you want to deploy, for example *process-app-start*.
-. Click *Build & Deploy*. 
+. Click *Build & Deploy*.
 +
 * If there is no container with the project name, a container with default values is automatically created.
-* If there already is a container with the project name, a dialogue appears. You can change the name of your container and its alias. When finished, click *Ok*.
+* If there already is a container with the project name, a window opens. You can change the name of your container and its alias. When finished, click *Ok*.
```
Comment (gsheldon) [inline docs/product-getting-started-guide/src/main/asciidoc/task_deploying-application.adoc]: "If there already is a container with the project name, a window opens". Suggest: "A window opens if a container with the project name already exists."? WDYT?

File: docs/product-getting-started-guide/src/main/asciidoc/task_business-rules.adoc (hunk 1 above)
Comment (michelehaglund) [reply docs/product-getting-started-guide/src/main/asciidoc/task_business-rules.adoc]: There are actually 2 intro lines like that in this topic. I simply removed them. The section title is really all that is needed. That's my version of minimalism :)

File: docs/product-getting-started-guide/src/main/asciidoc/task_business-rules.adoc (hunk 2 above)
Comment (michelehaglund) [reply docs/product-getting-started-guide/src/main/asciidoc/task_business-rules.adoc]: Corrected.

File: docs/product-getting-started-guide/src/main/asciidoc/task_process-forms.adoc [hunk 9]
```diff
@@ -6,81 +6,143 @@ The following chapter instructs you how to create forms for your business proces
 [float]
 == Prerequisites
 
-* Finished business process model of the started project. For further information, see <<_modeling_a_business_process>>.
+Finished business process model of the started project. For further information, see <<_modeling_a_business_process>>.
 
 == Automatic Form Generation
 
-{PRODUCT} enables you to automatically generate all forms. To generate forms for your business process:
+{PRODUCT} enables you to automatically generate all forms. For this business process you are: Creating the Applicant form, Creating the Property form, and Creating the Application form.
```
Comment (gsheldon) [inline docs/product-getting-started-guide/src/main/asciidoc/task_process-forms.adoc]: I'd suggest using bullet points for Creating the Applicant form, Creating the Property form, and Creating the Application form.

File: docs/product-getting-started-guide/src/main/asciidoc/task_process-forms.adoc [hunk 10]
```diff
@@ -6,81 +6,143 @@ The following chapter instructs you how to create forms for your business proces
...
-. Click the *Applicant* data object form. 
+. Log into Business Central. Click *Menu* -> *Projects*, then *process-app-start*.
+. Click the *Applicant* data object form.
 .. Remove the following fields:
 
 * *Address*
 * *Credit Rating*
 
-.. Click *Save*, then click *Save* to confirm, to save your changes.
+. Click *Save*, then click *Save* to confirm, to confirm your changes.
```
Comment (gsheldon) [inline docs/product-getting-started-guide/src/main/asciidoc/task_process-forms.adoc]: Remove extra "to confirm".

File: docs/product-getting-started-guide/src/main/asciidoc/task_process-forms.adoc [hunk 11]
```diff
@@ -6,81 +6,143 @@ The following chapter instructs you how to create forms for your business proces
...
 
 
-. Click the *Property* data object form. 
+. Click the *Property* data object form.
 .. Remove the following form field:
 
 * *Address of the Property*
 
-.. Click *Save*, then click *Save* to confirm, to save your changes.
+.. Click *Save*, then click *Save* to confirm, to confirm your changes.
```
Comment (gsheldon) [inline docs/product-getting-started-guide/src/main/asciidoc/task_process-forms.adoc]: Remove extra "to confirm".

File: docs/product-getting-started-guide/src/main/asciidoc/task_process-forms.adoc [hunk 12]
```diff
@@ -6,81 +6,143 @@ The following chapter instructs you how to create forms for your business proces
...
 
-. Click the *Application* data object form. 
+. Click the *Application* data object form.
 .. Remove the following form field with its subfield:
 
 * *Error details*
 
-.. Click *Save*, then click *Save* to confirm, to save your changes.
-.. Click the *process-app-start* label to return to the *Assets* view of the project. 
+.. Click *Save*, then click *Save* to confirm, to confirm your changes.
```
Comment (gsheldon) [inline docs/product-getting-started-guide/src/main/asciidoc/task_process-forms.adoc]: Remove extra "to confirm".

File: docs/product-getting-started-guide/src/main/asciidoc/task_process-forms.adoc [hunk 13]
```diff
@@ -93,36 +155,35 @@ Follow these steps to finish the process task forms:
 * *inlimit*
 * *incdownpayment*
 
-.. Click *Save*, then click *Save* to confirm, to save your changes.
-.. Click the *process-app-start* label to return to the *Assets* view of the project. 
+.. Click *Save*, then click *Save* to confirm, to confirm your changes.
```
Comment (gsheldon) [inline docs/product-getting-started-guide/src/main/asciidoc/task_process-forms.adoc]: Remove extra "to confirm".

File: docs/product-getting-started-guide/src/main/asciidoc/task_process-forms.adoc [hunk 14]
```diff
@@ -93,36 +155,35 @@ Follow these steps to finish the process task forms:
...
 
-. Click the *CorrectData-taskform* form. 
+. Click the *CorrectData-taskform* form.
 .. Remove the following subform:
 
 * *taskoutputApplication*
 
-.. Click *Save*, then click *Save* to confirm, to save your changes.
-.. Click the *process-app-start* label to return to the *Assets* view of the project. 
+.. Click *Save*, then click *Save* to confirm, to confirm your changes.
```
Comment (gsheldon) [inline docs/product-getting-started-guide/src/main/asciidoc/task_process-forms.adoc]: Extra "to confirm".

File: docs/product-getting-started-guide/src/main/asciidoc/task_process-forms.adoc [hunk 15]
```diff
@@ -93,36 +155,35 @@ Follow these steps to finish the process task forms:
...
+.. Click the *process-app-start* label to return to the *Assets* view of the project.
 
 . Click the *IncreaseDownPayment-taskform* form.
 .. Next to the *incdownpayment* field, click the edit icon (image:gsgEditBtn.png[]), then click *Edit*.
 +
-In the *Field Properties* dialogue, change the *Label* textbox to `Increase Down Payment`. Then click *Ok*.
-.. Click *Save*, then click *Save* to confirm, to save your changes.
-.. Click the *process-app-start* label to return to the *Assets* view of the project. 
+In the *Field Properties* window, change the *Label* textbox to `Increase Down Payment`. Then click *Ok*.
+.. Click *Save*, then click *Save* to confirm, to confirm your changes.
```
Comment (gsheldon) [inline docs/product-getting-started-guide/src/main/asciidoc/task_process-forms.adoc]: Extra "to confirm".

File: docs/product-getting-started-guide/src/main/asciidoc/task_process-forms.adoc [hunk 16]
```diff
@@ -93,36 +155,35 @@ Follow these steps to finish the process task forms:
...
+.. Click the *process-app-start* label to return to the *Assets* view of the project.
 
 . Click the *FinalApproval-taskform* form.
 .. Next to the *inlimit* field, click the edit icon (image:gsgEditBtn.png[]), then click *Edit*.
 +
-In the *Field Properties* dialogue, change the *Label* textbox to `In limit?`. Then click *Ok*.
+In the *Field Properties* window, change the *Label* textbox to `In limit?`. Then click *Ok*.
 
 .. Click *Save*, then click *Save* to confirm, to save your changes.
```
Comment (gsheldon) [inline docs/product-getting-started-guide/src/main/asciidoc/task_process-forms.adoc]: I don't know if this needs both "to confirm" and "to save your changes"? Does "then click Save to confirm and save your changes" still mean the same thing?

File: docs/product-getting-started-guide/src/main/asciidoc/task_process-forms.adoc [hunk 17]
```diff
@@ -93,36 +155,35 @@ Follow these steps to finish the process task forms:
...
+.. Click the *process-app-start* label to return to the *Assets* view of the project.
 
 . Click the *Qualify-taskform* process form.
 .. Next to the *inlimit* field, click the edit icon (image:gsgEditBtn.png[]), then click *Edit*.
 +
-In the *Field Properties* dialogue, change the *Label* textbox to `In limit?`. Then click *Ok*.
-.. Click *Save*, then click *Save* to confirm, to save your changes.
-.. Click the *process-app-start* label to return to the *Assets* view of the project. 
+In the *Field Properties* window, change the *Label* textbox to `In limit?`. Then click *Ok*.
+.. Click *Save*, then click *Save* to confirm, to confirm your changes.
```
Comment (gsheldon) [inline docs/product-getting-started-guide/src/main/asciidoc/task_process-forms.adoc]: Extra "to confirm".

File: docs/product-getting-started-guide/src/main/asciidoc/task_process-forms.adoc (hunk 16 above)
Comment (michelehaglund) [reply docs/product-getting-started-guide/src/main/asciidoc/task_process-forms.adoc]: Saw that one last night. Thought that I had caught it. Thanks for the catch.

**Pull request merged**